spec/index.bs

```diff
@@ -11255,9 +11257,8 @@ GPUQueue includes GPUObjectBase;
             **Returns:** {{undefined}}
 
             1. Let |sourceImage| be |source|.{{GPUImageCopyExternalImage/source}}
-            1. Run [=Check the usability of the image argument=](|sourceImage|).
+            1. Let |usability| be [=?=] [=check the usability of the image argument=](|source|).
```

Comment:
Technically this returns a value "synchronously" on the JS side, but the result is not actually used synchronously so under "[as-if](https://gpuweb.github.io/gpuweb/#w3c-conformant-algorithms)" I think it's fine.